src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-accent;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $color-white;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &:not(:nth-of-type(4n)) {
+    margin-right: $card-margin-right;
+  }
+
+  &:nth-of-type(-n + 4) {
+    margin-bottom: 46px;
+  }
+
+  &__img {
+    margin: 0 auto 40px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__cod {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    font-weight: $font-weight-normal;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: $font-weight-normal;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    &-info {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 16px;
+    }
+
+    &-caption {
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary;
+      font-weight: $font-weight-normal;
+      font-style: normal;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
```

Comment:
do not hardcode elements' width unless you have to. here the button's width needs to be 100% of the parent. simply use display block without having to set the width. The same with height

Reply:
I deleted width, but don't touched height. I think, it's not necessary, and i like to center text with display:flex.